### ISSUE TITLE
Unable to run Camunda-Modeler on Fedora 35

### ISSUE THREAD

**Issue by DanHamCIC** [camunda/camunda-modeler#2774]
__Describe the Bug__

I am unable to run Camunda Modeler 4.12.0 on Fedora 35 following the instructions on the website.

```
$ curl -SLO https://downloads.camunda.cloud/release/camunda-modeler/4.12.0/camunda-modeler-4.12.0-linux-x64.tar.gz
$ tar xzf camunda-modeler-4.12.0-linux-x64.tar.gz
$ cd camunda-modeler-4.12.0-linux-x64
$ ./camunda-modeler 
INFO app:main:bootstrap starting Camunda Modeler v4.12.0
INFO app:cli parsing [ '/home/dan/camunda-modeler-4.12.0-linux-x64/camunda-modeler' ] in '/home/dan/camunda-modeler-4.12.0-linux-x64'
INFO app:flags searching for flags.json in paths [
  '/home/dan/camunda-modeler-4.12.0-linux-x64/resources',
  '/home/dan/.config/camunda-modeler/resources'
]
INFO app:flags found []
INFO app:flags active {}
INFO app:error-tracking Cannot initialize Sentry:  Crash reports not enabled.
INFO app:plugins searching for plugins/*/index.js in paths [
  '/home/dan/camunda-modeler-4.12.0-linux-x64',
  '/home/dan/camunda-modeler-4.12.0-linux-x64/resources',
  '/home/dan/.config/camunda-modeler/resources',
  '/home/dan/.config/camunda-modeler'
]
INFO app:plugins found plug-in entries []
INFO app:plugins registered []
INFO app:main:bootstrap received ready
libva error: vaGetDriverNameByIndex() failed with unknown libva error, driver_name = (null)
...
libva error: vaGetDriverNameByIndex() failed with unknown libva error, driver_name = (null)
[5332:0221/151751.203749:FATAL:gpu_data_manager_impl_private.cc(415)] GPU process isn't usable. Goodbye.
Trace/breakpoint trap (core dumped)
```

This appears to be due to a problem with the version of electron that is being used and an update to glibc. This was fixed in electron with various PR's merged into the 12-x-y, 13-x-y branches (see https://github.com/electron/electron/pull/31091 and https://github.com/electron/electron/pull/31096).

__Workaround__

To get the current 4.12.0 version of Camunda Modeler to run I had to add the following flag:

```
$ ./camunda-modeler --disable-gpu-sandbox
```

The modeler window then appears.

__Alternative Fix__

I tried to compile camunda-modeler (from the tip of the develop branch) to see if that made any difference. Unfortunately it did not - the process didn't crash, but the Camunda Modeler window did not appear either. Currently this has electron set to `12.1.2` in the package.json file.

However, compiling again with the version of electron set to `17.0.1` allows me to run Camunda Modeler 5.0.0-alpha.1 without issue (and without the `--disable-gpu-sandbox` flag).

As an aside - 5.0.0 looks really nice! I like the improvements!

__Environment__

- OS: Fedora 35 (Workstation Edition) running GNOME
- Camunda Modeler Version: 4.12.0
- Execution Platform: N/A but Camunda Platform
- Installed plug-ins: None

**Comment by MaxTru** [comment]
Hi,

thanks for reporting and thanks for the feedback! Also great that you already found a workaround.

Bumping to latest electron is on our radar and we will follow-up. However, I cannot give a timeline.

See https://github.com/camunda/camunda-modeler/issues/2538.

Thanks.

**Comment by DanHamCIC** [comment]
> Bumping to latest electron is on our radar and we will follow-up. However, I cannot give a timeline.

Version 17.0.1 of electron works for me for version 5.0.0-alpha.1

However, when I tried to do the same for version 4.12.0 I got a build error - I checked out a new branch based off the v4.12.0 tag, updated the electron version and tried to build. That said, I'm an absolute beginner with this and it's 100% likely that I've done something wrong there. I fully expect your dev's would be able to get this going.

Understood on the time-line - I look forward to the 5.0.0 release!
